### PR TITLE
Add an option to supply custom_sample_info_table.

### PR DESCRIPTION
**Pull request by tneymanov**

### CODE REVIEW

File: gcp_variant_transforms/options/variant_transform_options.py [hunk 1]
```diff
@@ -658,19 +664,27 @@ def validate(self, parsed_args, client=None):
...
-
-    if not bigquery_util.table_exist(client, project_id, dataset_id,
-                                     sample_table_id):
+    if parsed_args.custom_sample_info_table:
+      sample_project_id, sample_dataset_id, sample_table_id = (
+          bigquery_util.parse_table_reference(
+              parsed_args.custom_sample_info_table))
+    else:
+      sample_project_id = project_id
+      sample_dataset_id = dataset_id
```
Comment (samanvp) [inline gcp_variant_transforms/options/variant_transform_options.py:673]: nit: Let's move these two lines to where `sample_table_id` is initiated.

**Review by samanvp — APPROVED** [review]
Thanks for clean PR.

File: gcp_variant_transforms/options/variant_transform_options.py (hunk 1 above)
Comment (tneymanov) [reply gcp_variant_transforms/options/variant_transform_options.py]: Done.

**Review by tneymanov — COMMENTED** [review]
Will run int tests and submit afterwards. Thanks.

**Pull request merged**